packages/constants/odf.ts

```diff
@@ -0,0 +1,8 @@
+import { TFunction } from 'i18next';
+
+export const diskTypeDropdownItems = (t: TFunction) =>
+  Object.freeze({
+    All: t('plugin__dfr-console~All'),
+    SSD: t('plugin__dfr-console~SSD / NVMe'),
+    HDD: t('plugin__dfr-console~HDD'),
+});
```

Comment:
why this file ??
it is not used anywhere right ??

Reply:
yes, it is not used anywhere so removed the file.